things/service/src/main/java/org/eclipse/ditto/things/service/persistence/actors/ThingSupervisorActor.java

```diff
@@ -258,7 +261,7 @@ protected CompletionStage<Object> askEnforcerChild(final Signal<?> signal) {
                     )
                     .orElseGet(() -> super.askEnforcerChild(thingCommandResponse).toCompletableFuture());
         } else {
-            return super.askEnforcerChild(signal);
+            return Patterns.ask(enforcerChild, signal, DEFAULT_LOCAL_ASK_TIMEOUT.multipliedBy(4L));
```

Comment:
I don't especially like this "magic number" `4L` hidden somewhere deep in the code.
However, also the constant `DEFAULT_LOCAL_ASK_TIMEOUT` is not very flexible (that was my bad I think) and I would not use it here ..

Best would be IMHO to make the "local ask timeout" configurable, so that `defaultLocalAskTimeout` in `AbstractPersistenceSupervisor` is read from configuration.

If you, e.g., need that to be higher in order to mitigate a performance/scaling bottleneck, you can e.g. increase it then very exactly for "things" service.

And you have the "truth" at a single place, the configuration - instead of the effective timeout for a specific case being multiplied somewhere in the code.